algorithmic_efficiency/workloads/imagenet/imagenet_pytorch/workload.py

```diff
@@ -28,7 +30,7 @@ def cycle(iterable):
       iterator = iter(iterable)
 
 
-class ImagenetPytorchWorkload(ImagenetWorkload):
+class ImagenetPytorchWorkload(BaseImagenetWorkload):
```

Comment:
Nitpick: I believe it should be `ImagenetWorkload` to be consistent with the other workloads.

However, this is probably just a temporary convention. Once we have multiple workloads all using ImageNet, we need to rethink this anyway. We could use something like `DatasetModelFramework` as our naming convention (e.g. `ImagenetResNetPytorch` in this case, although the `Framework` part might not be necessary.

Reply:
I think that it may be easier to remove the `Framework` from the name, because if we add it then we need to change the workload registry in `submission_runner.py` to have a jax and pytorch class name (it's just easier if they are the same). In that case, we need the `Base` in the name of the parent class, to avoid redefining the class (one of the linter tests complained about it IIRC). I'll change the other workload to have the `Base` convention.

Reply:
Agreed!